packages/components/src/Button.tsx

```diff
@@ -113,6 +113,22 @@ export const SecondaryButton = BaseButton.extend`
   }
 `;
 
+export const DarkButton = BaseButton.extend`
+  background-color: ${colors.primary.BLACK};
+  color: ${colors.basic.WHITE};
+  border: 1px solid ${colors.primary.BLACK};
+  &.active {
+    background-color: ${colors.accent.CIVIL_GRAY_1};
+    border: 1px solid ${colors.accent.CIVIL_GRAY_1};
+    color: ${colors.basic.WHITE};
+  }
+  &.active {
+    background-color: ${colors.accent.CIVIL_GRAY_1};
+    border: 1px solid ${colors.accent.CIVIL_GRAY_1};
+    color: ${colors.basic.WHITE};
+  }
```

Comment:
Should the second `&.active` be a different state? I see it happens on `SecondaryButton` also.

Reply:
Ah, good catch. I was copy-pasta-ing those definitions from `SecondaryButton`, but my guess is that they're supposed to be `:hover` definitions in there. I'm updating with that assumption, and we can address further if the design team has feedback.